packages/babel-parser/test/fixtures/experimental/discard-binding/using-variable-declarator/input.js

```diff
@@ -0,0 +1,34 @@
+{
+  // using 1 declarator
+  using void = f();
+}
+{
+  // using 2 declarators
+  using void = f(), void = g();
+}
+{
+  // using void declarator and normal declarator
+  using void = f(), x = g();
+}
+{
+  // using declarator in for-of
+  for(using void of []);
```

Comment:
Although the parser currently accepts this production, it is not allowed in the draft spec. I have filed an issue https://github.com/tc39/proposal-discard-binding/issues/14

Reply:
Yeah this seems like a spec bug.